checker/src/main/java/org/checkerframework/checker/resourceleak/MustCallInferenceLogic.java

```diff
@@ -29,8 +62,8 @@
  */
 public class MustCallInferenceLogic {
 
-  /** The set of owning fields. */
-  private final Set<VariableElement> owningFields = new HashSet<>();
+  /** The set of owning fields that are released within the enMethodElt element. */
```

Comment:
This comment doesn't make sense on its own: I don't know what the `enMethodElt` element is without referring to another field. Maybe something like "the set of owning fields that are released within the CFG currently under analysis" or something like that?

Reply:
@Nargeshdb You don't need to reply with "done".  You can just click "resolve conversation" if no further discussion is needed.  (That may not even be necessary, since a change to the relevant lines may cause GitHub to resolve the conversation.)  But please do comment when followup or further discussion is necessary.  Thanks!

Reply:
Thanks for the advice, I'll keep this in mind and resolve these conversations in this PR.